ui/admin/app/routes/scopes/scope/session-recordings/index.js

```diff
@@ -125,6 +128,31 @@ export default class ScopesScopeSessionRecordingsIndexRoute extends Route {
     );
   }
 
+  /**
+   * Sets doSessionRecordingsExist to true if there are any session recordings.
+   * @param {string} scope_id
+   * @param {number} totalItems
+   * @returns {Promise<boolean>}
+   */
+  async getDoSessionRecordingsExist(scope_id, totalItems) {
```

Comment:
What was the purpose again of using this method? It looks like we still have to make the request for all sessions regardless

Reply:
Good question, the query for all session recordings only happens on initial load or refresh. There was a bug where if on initial load if there were no session-recordings then on a subsequent load when I do have session-recordings they would not show up.

Reply:
Could we fix that bug by checking the length of the response in the `getAllSessionRecordings` method? If the length is 0, we set `this.allSessionRecordings = null`?  That would fix the bug and remove the need for this check as well.

Reply:
Then there could be the case where length was >0 but the session recordings were deleted and user navigates back to the list page and it'll show the filters and search box but no table. I'll also test this out to make sure.